src/app/workflow/workflow.component.html

```diff
@@ -42,21 +42,27 @@ <h2 class="text-center">The workflow linked to could not be found!</h2>
       </ul>
     </div>
   </div>
-  <div class="row m-1" *ngIf="!showRedirect">
+  <div class="row m-1" *ngIf="!showRedirect" fxLayout.lt-sm="column">
     <ngx-json-ld [json]="schema"></ngx-json-ld>
-    <div class="col-md-7">
+    <div fxFlex="80">
       <h3 id="workflow-path" data-cy="workflowTitle" fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="1rem">
-        <span id="verifiedIcon" *ngIf="(extendedWorkflow$ | async)?.versionVerified">
-          <a [href]="getVerifiedLink()" class="verified-check">
-            <mat-icon matTooltip="Verified">done</mat-icon>
-          </a>
+        <span>
+          <img src="../../assets/svg/workflow-circle.svg" alt="workflow icon" />
         </span>
         <span>
           {{ title
           }}<span *ngIf="workflow?.workflowVersions.length > 0"
-            >:<span class="ds-green">{{ selectedVersion?.name }}</span></span
+            >:<span class="ds-steelblue">{{ selectedVersion?.name }}</span></span
```

Comment:
if I make each span on their own lines it seems like the test doesn't detect the `title+":"+selectedVersion.name `correctly?